canonical-web-and-design/www.ubuntu.com
# Core 18 security takeover and landing page

## Done

Add Core 18 security takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/core18-security](http://0.0.0.0:8001//engage/core18-security)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to home page and refresh the page until you see the Core 18 takeover. After you check it, click on "Watch the webinar" button and check the landing page
- Make sure it complies with the [brief](https://docs.google.com/document/d/1i_Zo_IdVsGJlMtDKXQh77H_LtbNyS4_vie6vBTHnxbw/edit#)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/web-squad/issues/1170) and [more design](https://github.com/canonical-web-and-design/web-squad/issues/1178)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1178
